release_notes.md

```diff
@@ -10,3 +12,6 @@
 - By default, `IDurableEntityClient.ListEntitiesAsync` no longer returns deleted entities.
```

Comment:
If there are breaking changes pending, should we also remove the `StandardConnectionStringProvider` and `WebJobsConnectionStringProvider`?

Reply:
I'll elaborate more here about them being deprecated